dapp/src/pages/DashboardPage/AcrePointsCard.tsx

```diff
@@ -39,18 +47,21 @@ export default function AcrePointsCard(props: CardProps) {
     >
       <CardHeader p={0} mb={2}>
         <TextMd fontWeight="bold" color="grey.700">
-          Acre points balance
+          Your Acre points balance
         </TextMd>
       </CardHeader>
 
       <CardBody p={0}>
-        <UserDataSkeleton>
+        <Skeleton isLoaded={!isLoading}>
```

Comment:
I think we should use `UserDataSkeleton` here instead of `Skeleton`. Note that this acre points data appears earlier than the rest. See the video below:

https://github.com/user-attachments/assets/47faa4d1-ffb8-4494-ad3b-fc0a989e3817

The `UserDataSkeleton` will show the components when all the data for `userKeys` are fetched.

https://github.com/thesis/acre/blob/66e6dd37db8c53a74b4b3c46e65faea6e8b193c0/dapp/src/hooks/useIsFetchedWalletData.ts#L10-L12

https://github.com/thesis/acre/blob/2254b7ef7befa266974532c8510ab04ea2400e65/dapp/src/constants/queryKeysFactory.ts#L1-L6